lib/connectors/mail.js

```diff
@@ -148,23 +152,23 @@ Mailer.send = function(options, fn) {
   }
 
   if (debug.enabled || settings && settings.debug) {
-    console.log('Sending Mail:');
+    g.log('Sending Mail:');
     if (options.transport) {
-      console.log('\t TRANSPORT:', options.transport);
+      g.log('\t TRANSPORT:', options.transport);
     }
-    console.log('\t TO:', options.to);
-    console.log('\t FROM:', options.from);
-    console.log('\t SUBJECT:', options.subject);
-    console.log('\t TEXT:', options.text);
-    console.log('\t HTML:', options.html);
+    g.log('\t TO:', options.to);
+    g.log('\t FROM:', options.from);
+    g.log('\t SUBJECT:', options.subject);
+    g.log('\t TEXT:', options.text);
+    g.log('\t HTML:', options.html);
```

Comment:
I think the capital letters should be translated with the exception of {{HTML:}}. The order probably doesn't matter here.